aiida/orm/nodes/data/data.py

```diff
@@ -25,20 +25,15 @@
 
 class Data(Node):
     """
-    This class is base class for all data objects.
+    The base class for all Data nodes.
 
-    Specifications of the Data class:
-    AiiDA Data objects are subclasses of Node and should have
-
-    Multiple inheritance must be supported, i.e. Data should have methods for
-    querying and be able to inherit other library objects such as ASE for
+    AiiDA Data classes are subclasses of Node and must support multiple inheritance,
```

Comment:
I realize that you just rephrased the old sentence, but to me it does not make a lot sense still. I don't see how "multiple inheritance" has anything to do with being queryable or support construction of other library definitions. I think definitely the latter part should not be here. It should be up to the plugin developer to decide what its interface supports.

Reply:
hehe, similar questions here - I just started to modify this in order to get some feedback on this.

So, I propose to keep only `AiiDA Data classes are subclasses of Node and must support multiple inheritance.`